judge/highlight_code.py

```diff
@@ -23,15 +16,13 @@ def _wrap_code(inner):
     def highlight_code(code, language, cssclass=None):
         return _make_pre_code(code)
 else:
-    class HtmlCodeFormatter(pygments.formatters.HtmlFormatter):
-        def wrap(self, source, outfile):
-            return self._wrap_div(self._wrap_pre(_wrap_code(source)))
-
     def highlight_code(code, language, cssclass='codehilite'):
         try:
             lexer = pygments.lexers.get_lexer_by_name(language)
         except pygments.util.ClassNotFound:
             return _make_pre_code(code)
 
         # return mark_safe(pygments.highlight(code, lexer, HtmlCodeFormatter(cssclass=cssclass, linenos='table')))
```

Comment:
Delete this. I don't think DMOJ users are ready for `linenos='table'`.

Reply:
Done